changes/ce/fix-10154.en.md

```diff
@@ -0,0 +1,8 @@
+Change the default `resume_interval` for bridges and connectors to be
```

Comment:
Since this is already in `master`, should I keep this changelog entry pointing to that PR?  Or change its number to the present one?